chore: update project for app settings configuration first (#25)
slack-samples/bolt-python-custom-function-template: README.md

# Bolt for Python Custom Function Template

> This is a Bolt for Python template app used to build custom functions in

Reviewer: ```suggestion
This is a Bolt for Python template app used to build custom functions for use in in
```

project require that the workspace be part of
[a Slack paid plan](https://slack.com/pricing).**

### Developer Program

Join the [Slack Developer Program](https://api.slack.com/developer-program) for

> exclusive access to beta features, tooling, and resources created to help

Reviewer: ```suggestion
exclusive access to sandbox environments for building and testing your apps, beta features, tooling, and resources created to help
```

developers build and grow.

## Installation

### Create a Slack App

1. Open [https://api.slack.com/apps/new](https://api.slack.com/apps/new) and
   choose "From an app manifest"
2. Choose the workspace you want to install the application to
3. Copy the contents of [manifest.json](./manifest.json) into the text box that
   says `*Paste your manifest code here*` (within the JSON tab) and click _Next_
4. Review the configuration and click _Create_

> 5. Click _Install to Workspace_ and _Allow_ on the screen that follows. You'll

Reviewer: The button now doesn't always say Install to Workspace.

```suggestion
5. Click the _Install_ button and _Allow_ on the screen that follows. You'll
```

>    then be redirected to the App Configuration dashboard.

Reviewer: ```suggestion
   then be redirected to the App Settings dashboard.
```

### Environment Variables

Before you can run the app, you'll need to store some environment variables.

> 1. Open your apps configuration page from this list, click **OAuth &

Reviewer: ```suggestion
1. Open your apps setting page from this list, click **OAuth &
```

whoop found another!

Author: Good catch 💯 I should probably update this in all our other samples 🤔

   Permissions** in the left hand menu, then copy the Bot User OAuth Token. You
   will store this in your environment as `SLACK_BOT_TOKEN`.
2. Click ***Basic Information** from the left hand menu and follow the steps in
   the App-Level Tokens section to create an app-level token with the
   `connections:write` scope. Copy this token. You will store this in your
   environment as `SLACK_APP_TOKEN`.

```zsh
# Replace with your app token and bot token
export SLACK_BOT_TOKEN=<your-bot-token>
export SLACK_APP_TOKEN=<your-app-token>
```

### Local Project

```zsh
# Clone this project onto your machine
git clone https://github.com/slack-samples/bolt-python-custom-function-template.git

# Change into this project directory
cd bolt-python-custom-function-template

# Setup your python virtual environment
python3 -m venv .venv
source .venv/bin/activate

# Install the dependencies
pip3 install -r requirements.txt

# Start your local server
python3 app.py
```

### Linting

Run flake8 and black for linting and code formatting:

```zsh
# Run ruff from root directory for linting
ruff check

# Run ruff from root directory for code formatting
ruff format
ruff check --fix
```

> ## Using Functions in Workflow Builder

Reviewer: ```suggestion
## Using Steps in Workflow Builder
```
